features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -70,6 +87,69 @@ internal class DrawableUtils(
         return bitmapWrapper.createScaledBitmap(bitmap, width, height, false)
     }
 
+    internal interface ResizeBitmapCallback {
+        fun onSuccess(bitmap: Bitmap)
+        fun onFailure()
+    }
+
+    private fun executeRunnable(runnable: Runnable) {
```

Comment:
there is already an extension method `executeSafe` that you can use in the `ConcurrencyExt` class

Reply:
I believe there is no need to mock `executeSafe` in tests, because it is an extension method - it will be called already on the mock of the executor and the only other mock it needs it is the logger. So I guess the reason to not use it here is just the need to provide the logger which is not here yet.

Reply:
I spoke with Xavier about how it would be possible to test while using this extension. He suggested a solution so I will push executeSafe with the next changes and remove executeRunnable